troubleshooting/extra-android-permissions-always-set.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Extra Android permissions always set

```

Comment:
Title capitalization and eventually change **set** with **enabled**
For example: **Extra Android Permissions Always Enabled**